src/components/Header.js

```diff
@@ -3,12 +3,11 @@ import { Link } from 'react-router-dom';
 
 const Header = () => (
```

Comment:
- [ ] You should make your app looks exactly like the provided design, kindly consider adding the profile icon on the top right of the navigation
- [ ] To make your page closer to the provided design kindly consider adding the same background color for the navigation and using the same margin/padding so your app looks identical to the zeplin design

|Your app|Zeplin design|
|-|-|
|![Screenshot_20221110_210140](https://user-images.githubusercontent.com/25227022/201194996-951fa7d6-e798-4d88-b50e-4e7830c18838.png)|![Screenshot_20221110_210233](https://user-images.githubusercontent.com/25227022/201195023-b9e7f12e-cdea-4b1d-8b46-e50186e9df04.png)|